### PR TITLE
Support `netstandard1.3` using MailKit

### PR DESCRIPTION
**Pull request by adamchester**
Opening PR for feedback.

### CODE REVIEW

**Comment by nblumhardt** [comment]
Looking great!

File: src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            foreach (var logEvent in events)
+            {
+                _textFormatter.Format(logEvent, payload);
+            }
+
+            var mailMessage = CreateMailMessage(payload.ToString());
+
+            try
+            {
+                using (var smtpClient = OpenConnectedSmtpClient())
```
Comment (adamchester) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Probably needs async

Comment (nblumhardt) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Only one of the two `EmitBatch()` methods needs to be overridden - if the API supports `async` then flowing it throughout via the other overidde below should be the way to go. Is that what you're thinking?

Comment (adamchester) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Yep oops, wrong line, it should have been the other async method

Comment (adamchester) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Should be async

Comment (adamchester) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Wow that's really weird, a GitHub bug on the line comment!

Comment (nblumhardt) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Planning to push this one forwards? :-)

Comment (adamchester) [inline src/Serilog.Sinks.Email/Sinks/Email/MailKitEmailSink.cs]: Makes sense, lets hit the switch

**Pull request merged**